src/main/resources/org/csanchez/jenkins/plugins/kubernetes/pipeline/scripts/kill-processes-with-cookie.ps1

```diff
@@ -0,0 +1,32 @@
+param(
+    [string]$cookie,
+	[string]$csFile
+)
+
+Add-Type -Path $csFile
+
+$failed = $false
+Get-Process | ForEach-Object {
+        $id = $_.Id
+        try {
+    		$envBlock = [ProcessEnvironmentReader]::ReadEnvironmentBlock($id)
+            if ($envBlock.Contains("JENKINS_SERVER_COOKIE=$($cookie)")) {
+            	Write-Host "Killing $($_.ProcessName) (ID: $($_.Id)) - JENKINS_SERVER_COOKIE matches"
```

Comment:
Should I remove this? It is not gonna be visible to the end user but may help on local runsa nd debugging